### ISSUE TITLE
--spacesInImportCurlyBraces true will add spaces everywhere

### ISSUE THREAD

**Issue by schmitch** [olafurpg/scalafmt#424]
This template isn't a strict requirement to open issues, but please try to provide as much information as possible.

**Version**: 0.3.1
**Integration**: intellij, sbt, cli
**Configuration**: --spacesInImportCurlyBraces true

```scala
import play.api.libs.json.{JsObject, JsString, Json}

object Hase {

    def name(xs: OtherObject): String = {
        s"Hallo ${xs.name}"
    }
    
}
```

Scalafmt:
```scala
import play.api.libs.json.{ JsObject, JsString, Json }

object Hase {

    def name(xs: OtherObject): String = {
        s"Hallo ${ xs.name }"
    }
    
}
```

I would like the input to look like this:
```scala
import play.api.libs.json.{ JsObject, JsString, Json }

object Hase {

    def name(xs: OtherObject): String = {
        s"Hallo ${xs.name}"
    }
    
}
```

**Comment by olafurpg** [comment]
My guess is that this is caused by a small bug in spacesInImportCurlyBraces. I don't personally use this flag so it would be great if someone who does use the flag is willing to take a shot at this.

From my comment on gitter:
> the tutorial explains how to trace that space https://github.com/olafurpg/scalafmt#tutorial-add-spacesinparentheses-flag
> that'll pinpoint a line number in Router.scala where a Space is returned instead of a NoSplit

**Comment by olafurpg** [comment]
Fixed in #454
